main.go

```diff
@@ -167,16 +167,19 @@ func main() {
 
 		cm, err := util.GetCmOfMapCs(bs.Reader)
 		if err != nil {
-			// Create new common-service-maps
-			if errors.IsNotFound(err) {
-				klog.Infof("Creating common-service-maps ConfigMap in kube-public")
-				if err = bs.CreateCsMaps(); err != nil {
-					klog.Errorf("Failed to create common-service-maps ConfigMap: %v", err)
+			// Skip common-service-maps creating when installing in AllNamespace Mode
+			if bs.CSData.WatchNamespaces != "" {
```

Comment:
I am wondering if we could place this condition before getting ConfigMap?

The reason for this is to avoid unnecessary API Call given we are not worried about ConfigMap at all when it is deployed in All Namespaces Mode